### PR TITLE
Adding SpaceVim to Applications

### PR DESCRIPTION
**Pull request by ionlights**
[SpaceVim][sv] is a `vim` distribution that tries to mimic Spacemacs, but for `vim`. Configurations are stored in either:
1. `~/.SpaceVim.d/`
1. `$SPACEVIMDIR` ([see ref][sv-config])

The default is `~/.SpaceVim.d/`, so that's what makes sense for Mackup to support.

[sv]: https://spacevim.org
[sv-config]: https://spacevim.org/documentation/#custom-configuration

### CODE REVIEW

**Comment by lra** [comment]
Can you follow https://github.com/lra/mackup/blob/master/.github/CONTRIBUTING.md ?
Thanks

**Comment by ionlights** [comment]
Migrated computers which led to some issues with upstreaming just from my original branch – I can revert if necessary.

Otherwise, the changes from CONTRIBUTING should be there. 🙂

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,10 +2,12 @@
 
 ## WIP
 
+- Added support for SpaceVim (via @ionlights)
 - Added support for clashX (via @awkj)
 - Added support for Brave (via @cbenv)
 - Added support for Nushell (via @leesiongchan)
 - Updated support for Clipy (via @jclerc)
+- Added support for aerc (via @Crocmagnon)
```
Comment (lra) [inline CHANGELOG.md:10]: I don't think this is supposed to be part of this PR

**Comment by ionlights** [comment]
Sorry, needed to revert and overwrite changes. Force-pushing to my own repo seemed to be the simplest. Unsure if there's "a better way" – if so, could you point me in the right direction and I'll do that? 🙂

**Pull request merged**